### DISCUSSION TITLE
Student developer pack does not let me re-submit: Your uploaded proof has already been used previously.

### DISCUSSION THREAD

**Discussion by Divyadedestroyer** [community/community#59754] [GitHub Education]
### Hi everyone,

Hello everyone, 

I recently joined Github and I was trying to get the student developer pack. I had not updated my user name and bio and my request got rejected. 
Now that I am reapplying, I see the request getting rejected once more, for this reason: Your uploaded proof has already been used previously, which is highly frustrating.

Of course, it would have been the same ID card that I used previously for the same account, which is why I am using it :( 
Makes no sense, how am I supposed to get verified now?

**Comment by GhostOf0days** [comment]
You could upload a screenshot of your cost of attendance for the upcoming semester that has your school name in it.
